webcompat/views.py

```diff
@@ -102,7 +103,7 @@ def login():
             # manually set the referer so we know where to come back to
             # when we return from GitHub
             set_referer(request)
-            return github.authorize('public_repo')
+            return github.authorize()
     else:
```

Comment:
This is no longer needed as scopes are set during app creation/installation (and we don't really ask for any permissions apart from "act on your behalf").